src/main/java/seedu/address/logic/BudgetCalculationManager.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address.logic;
+
+import java.util.List;
+
+import seedu.address.model.Model;
+
+import seedu.address.model.clubbudget.FinalClubBudget;
+
+/**
+ * Checks whether the FinalBudgetsBook is empty to determine whether budgets have already been calculated
+ */
+
+public class BudgetCalculationManager {
+    private static boolean haveBudgetsBeenCalculated = false;
+    BudgetCalculationManager() {}
+
+    public static boolean getHaveBudgetsBeenCalculated(Model model) {
```

Comment:
Since you already have a constructor for BudgetCalculationManager on line 15, you can remove the static for this getHaveBudgetsBeenCalculated method and just instantiate a BudgetCalculationManager  object and then call object.getHaveBudgetsBeenCalculated() method.